Client/Frontend/Browser/Tabs/Middleware/TabManagerMiddleware.swift

```diff
@@ -11,6 +11,10 @@ class TabManagerMiddleware {
     var tabManagers: [WindowUUID: TabManager] = [:]
     var selectedPanel: TabTrayPanelType = .tabs
 
+    var normalTabsCount: String {
```

Comment:
nit for the name of that variable. I thought it would be an `Int` when reading it. Either `normalTabsCountText` or `normalTabsCountString` ? Not sure what's the best name there
```suggestion
    var normalTabsCountText: String {
```

Reply:
Sounds good I will update it in my next PR since the build is green and ready to merge